src/models/column.js

```diff
@@ -3,6 +3,7 @@ import mongoose from 'mongoose';
 const { Schema } = mongoose;
 
 const taskSchema = new Schema({
+  color: { type: String, default: '#484848' },
```

Comment:
I think #484848 is quite too dark color. Maybe we should use just white color? (#ffffff)